application/src/main/java/org/togetherjava/tjbot/commands/system/BotCore.java

```diff
@@ -268,6 +271,21 @@ public void onSelectMenuInteraction(SelectMenuInteractionEvent event) {
             .execute(() -> forwardComponentCommand(event, UserInteractor::onSelectionMenu));
     }
 
+    @Override
+    public void onModalInteraction(final ModalInteractionEvent event) {
+        logger.debug("Received modal event '{}' (#{}) on guild '{}'", event.getModalId(),
+                event.getId(), event.getGuild());
+        COMMAND_SERVICE.execute(() -> {
+            ComponentId componentId = handleGetComponentId(event, event.getModalId());
```

Comment:
the method returns nullable and u use this without any checks

Reply:
fixed